OloEngine/tests/Rendering/BoundingVolumeTest.cpp

```
@@ -0,0 +1,232 @@
+// =============================================================================
+// BoundingSphere Tests
+// =============================================================================
+
+TEST(BoundingSphere, ConstructFromCenterRadius)
+{
+    BoundingSphere sphere(glm::vec3(1.0f, 2.0f, 3.0f), 5.0f);
+    EXPECT_EQ(sphere.Center, glm::vec3(1.0f, 2.0f, 3.0f));
+    EXPECT_FLOAT_EQ(sphere.Radius, 5.0f);
+}
+
+TEST(BoundingSphere, ContainsCenter)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)